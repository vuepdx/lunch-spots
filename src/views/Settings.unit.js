import settings from './Settings'

describe('@views/Settings', () => {
  const wrapper = mount(settings, createComponentMocks())
  it('is a valid view', () => {
    expect(settings).toBeAComponent()
    expect(wrapper).toMatchSnapshot()
  })
})
