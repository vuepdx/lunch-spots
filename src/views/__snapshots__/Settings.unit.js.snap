// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@views/Settings is a valid view 1`] = `
<div class="container fill-height container--fluid">
  <div class="row align-center justify-center">
    <div class="grow text-center col">
      <div class="v-card v-sheet theme--light">
        <div class="v-card__text">
          <h2>Location Settings</h2>
          <div class="v-input theme--light v-text-field">
            <div class="v-input__control">
              <div class="v-input__slot">
                <div class="v-text-field__slot"><label for="input-4" class="v-label theme--light" style="left: 0px; position: absolute;">Search Location</label><input id="input-4" type="text"></div>
              </div>
              <div class="v-text-field__details">
                <div class="v-messages theme--light">
                  <div class="v-messages__wrapper"></div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
